pkg/cli/admin/ocpcertificates/regeneratemco/command.go

```diff
@@ -0,0 +1,128 @@
+package regeneratemco
+
+import (
+	"context"
+	"time"
+
+	"github.com/openshift/library-go/pkg/operator/certrotation"
+	"github.com/spf13/cobra"
+
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+const (
+	oneYear    = 365 * 24 * time.Hour
+	caExpiry   = 10 * oneYear
+	caRefresh  = 9 * oneYear
+	keyExpiry  = caExpiry
+	keyRefresh = caRefresh
+
+	mcoNamespace   = "openshift-machine-config-operator"
+	mapiNamespace  = "openshift-machine-api"
+	controllerName = "OCMachineConfigServerRotator"
+	mcsName        = "machine-config-server"
+
+	// mcsTlsSecretName is created by the installer and is not owned by default
+	mcsTlsSecretName = mcsName + "-tls"
+
+	// newMCSCASecret is the location of the CA after rotation
+	newMCSCASecret = "machine-config-server-ca"
+	userDataKey    = "userData"
+
+	// mcoManagedWorkerSecret is the unused, MCO-managed stub ignition for workers
+	mcoManagedWorkerSecret = "worker-user-data-managed"
+	// mcoManagedMasterSecret is the unused, MCO-managed stub ignition for masters
+	mcoManagedMasterSecret = "master-user-data-managed"
+	// mcsLabelSelector is used to select the MCS pods
+	mcsLabelSelector = "k8s-app=machine-config-server"
+
+	// ign* are for the user-data ignition fields
+	ignFieldIgnition = "ignition"
+	ignFieldSecurity = "security"
+	ignFieldTLS      = "tls"
+	ignFieldCA       = "certificateAuthorities"
+	ignFieldSource   = "source"
+	ignDataPrefix    = "data:text/plain;charset=utf-8;base64,"
+)
+
+var (
+	regenerateMCOLong = templates.LongDesc(`
+		Regenerate the Machine Config Operator certificates for an OCP v4 cluster.
+
+		More information about these certificates are in the product documentation;
+		visit [docs.openshift.com](https://docs.openshift.com)
+		and select the version of OpenShift you are using.
```

Comment:
Remove this part please. From reading this, I do not know where to look, still. Reading the docs if something's unclear is what we'd expect from our users even if we don't mention it here.